x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.ts

```diff
@@ -100,6 +116,86 @@ const ExecutionLogSortFields: Record<string, string> = {
   num_active_alerts: 'ruleExecution>numActiveAlerts',
   num_recovered_alerts: 'ruleExecution>numRecoveredAlerts',
   num_new_alerts: 'ruleExecution>numNewAlerts',
+  num_errored_actions: 'numErroredActions>_count',
+};
+
+export const getExecutionKPIAggregation = (filter?: IExecutionLogAggOptions['filter']) => {
+  const dslFilterQuery: estypes.QueryDslBoolQuery['filter'] = buildDslFilterQuery(filter);
+
+  return {
+    excludeExecuteStart: {
+      filter: {
+        bool: {
+          must_not: [
+            {
+              term: {
+                'event.action': 'execute-start',
+              },
+            },
+          ],
+        },
+      },
+      aggs: {
+        actionExecution: {
+          filter: {
+            bool: {
+              ...(dslFilterQuery ? { filter: dslFilterQuery } : {}),
+              must: [getProviderAndActionFilter('alerting', 'execute')],
```

Comment:
I think this need to be `actions` not `alerting`. When I have some successful rule executions and 2 failed rule executions with no errored actions, I see 2 under errored actions.

Reply:
I fixed that when I did the unit test!